classes/class-camptix-payment-method-instamojo.php

```diff
@@ -240,162 +255,164 @@ function payment_notify() {
+
+		$instamojo_key   = $this->options['Instamojo-Api-Key'];
+		$instamojo_token = $this->options['Instamojo-Auth-Token'];
+		$instamojo_salt  = $this->options['Instamojo-salt'];
+
+		$order_amount = $order['total'];
 		if ( isset( $this->camptix_options['event_name'] ) ) {
 			$productinfo = $this->camptix_options['event_name'];
-		}else{
-			$productinfo = 'Ticket for Order - '.$payment_token;
+		} else {
+			$productinfo = 'Ticket for Order - ' . $payment_token;
 		}
 
 		$attendees = get_posts(
 			array(
-				'post_type'		=> 'tix_attendee',
-				'post_status'	=> 'any',
-				'orderby' 		=> 'ID',
-				'order'			=> 'ASC',
-				'meta_query' => array(
+				'post_type'   => 'tix_attendee',
+				'post_status' => 'any',
+				'orderby'     => 'ID',
+				'order'       => 'ASC',
+				'meta_query'  => array(
 					array(
-						'key' => 'tix_payment_token',
+						'key'     => 'tix_payment_token',
 						'compare' => '=',
-						'value' => $payment_token
-					)
-				)
+						'value'   => $payment_token,
+					),
+				),
 			)
 		);
-	
+
 		foreach ( $attendees as $attendee ) {
-				$tix_id = get_post( get_post_meta( $attendee->ID, 'tix_ticket_id', true ) );
+			$tix_id             = get_post( get_post_meta( $attendee->ID, 'tix_ticket_id', true ) );
 			$attendee_questions = get_post_meta( $attendee->ID, 'tix_questions', true ); // Array of Attendee Questons
-			
+
 			$email = $attendee->tix_email;
-			$name = $attendee->tix_first_name.' '.$attendee->tix_last_name;
+			$name  = $attendee->tix_first_name . ' ' . $attendee->tix_last_name;
 
 		}
 
 
 		$info       = $this->get_order( $payment_token );
 		$extra_info = array(
-			'phone'          => get_post_meta( $info['attendee_id'], 'tix_phone', true ),
-			
+			'phone' => get_post_meta( $info['attendee_id'], 'tix_phone', true ),
+
 		);
-		
-
-			$ch = curl_init();
-
-			$url = $this->options['sandbox'] ? 'https://test.instamojo.com/api/1.1/payment-requests/' : 'https://www.instamojo.com/api/1.1/payment-requests/';
-
-			curl_setopt($ch, CURLOPT_URL, $url);
-			curl_setopt($ch, CURLOPT_HEADER, FALSE);
-			curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
-			curl_setopt($ch, CURLOPT_FOLLOWLOCATION, TRUE);
-			curl_setopt($ch, CURLOPT_HTTPHEADER,
-			            array("X-Api-Key:$instamojo_key",
-			                  "X-Auth-Token:$instamojo_token"));
-
-
-			$payload = Array(
-			    'purpose' => $productinfo,
-			    'amount' => $order_amount,
-			    'phone' => $extra_info['phone'],
-			    'buyer_name' =>$name,
-			    'redirect_url' => $return_url,
-			    'send_email' => false,
-			    'webhook' => $notify_url,
-			    'send_sms' => false,
-			    'email' => $email,
-			    'allow_repeated_payments' => false
-			);
-			
 
-			curl_setopt($ch, CURLOPT_POST, true);
-			curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($payload));
-			$response = curl_exec($ch);
-			curl_close($ch); 
 
-			$json_decode = json_decode($response , true);
-			
-			if($json_decode['success']){
+		$ch = curl_init();

```

Comment:
I have mentioned @sanyogs we can wait for him to replay, presently there is a branch conflict, will review and merge; @sanyogs can work on the instamojo part of it. Or if I get the time before Tue, 25-Jul-2017; I will git it a try if not done.